docs/source/backends/airos.rst

```diff
@@ -0,0 +1,371 @@
+=============
+AirOS Backend
+=============
+
+.. include:: ../_github.rst
+
+The ``AirOs`` backend allows to generate AirOS v8.3 compatible configurations.
+
+.. warning::
+   This backend is experimental, please read this document carefully to prevent misconfiguration problems
```

Comment:
I would prefer the following:

```
This backend is in experimental stage: it may have bugs and it will
receive backward incompatible updates in during the first 6 months
of development (starting from September 2017).
Early feedback and contributions are very welcome and will help
to stabilize the backend faster.
```

I want people to know we will probably change the code radically in the initial phases after testing it in the real world (as it happened with the `OpenWrt` backend.